Warning: no DISPLAY environment variable.
--No graphics will be displayed.
Traceback (most recent call last):
  File "sequential_inference_17params.py", line 468, in <module>
    main(sys.argv[1:])
  File "sequential_inference_17params.py", line 195, in main
    trace_torch = torch.tensor(trace.values, dtype = torch.float32)
NameError: name 'trace' is not defined
